### PR TITLE
Update zmenu layout in genome browser

### PR DESCRIPTION
**Pull request by imransl**
## Type

- [ ] Bug fix
- [ ] New feature
- [x] Improvement to existing feature

## Related JIRA Issue(s)
[ENSWBSITES-1337](https://www.ebi.ac.uk/panda/jira/browse/ENSWBSITES-1337)

## Description
The zmenu layout design was recently changed (https://xd.adobe.com/view/2f3a9ffe-8e84-47fa-a796-54b337e2aa70-7a69/). The following had to be updated in the zmenu layout:

- move 'Download' link to left of panel
- add chevron to 'Download' link (same as EV transcript info panel)
- move 'View in' component to right of panel
- always show all relevant apps (GB & EV for MAP) for focus object

## Deployment URL
https://update-zmenu-layout.review.ensembl.org/

## Views affected
Genome browser -> zmenu

### Other effects
- [ ] I have checked any requirements for Google Analytics
- [x] I have created any required tests
- [ ] The PR has as its final commit a WASM/JS update commit if any Rust files were updated.

### CODE REVIEW

File: src/ensembl/src/shared/components/toolbox/toolbox-expandable-content/ToolboxExpandableContent.tsx [hunk 1]
```diff
@@ -15,10 +15,9 @@
  */
 
 import React, { useState, useContext, ReactNode } from 'react';
-import classNames from 'classnames';
 import noop from 'lodash/noop';
 
-import CloseButton from 'src/shared/components/close-button/CloseButton';
+import Chevron from 'src/shared/components/chevron/Chevron';
```
Comment (azangru) [inline src/ensembl/src/shared/components/toolbox/toolbox-expandable-content/ToolboxExpandableContent.tsx:20]: What if we used the `ShowHide` component for this?

Comment (azangru) [reply src/ensembl/src/shared/components/toolbox/toolbox-expandable-content/ToolboxExpandableContent.tsx:20]: Like so:

```tsx
 
  return (
    <ShowHide label={props.label} onClick={handleClick} isExpanded={isExpanded} />
  );
```

Comment (imransl) [reply src/ensembl/src/shared/components/toolbox/toolbox-expandable-content/ToolboxExpandableContent.tsx:20]: @azangru I wonder if I misunderstood this. Are you suggesting to use `ShowHide` in `ToolboxExpandableContent.tsx` or in `ZmenuAppLinks.tsx`?

Comment (azangru) [reply src/ensembl/src/shared/components/toolbox/toolbox-expandable-content/ToolboxExpandableContent.tsx:20]: in `ToolboxExpandableContent.tsx`. Right where you are using the `Chevron`.

Comment (imransl) [reply src/ensembl/src/shared/components/toolbox/toolbox-expandable-content/ToolboxExpandableContent.tsx]: @azangru Just pushed that change. FIgured out this morning that you might have said about the former. Yesterday, when I said I tried using `ShowHide`, I meant that for `ZmenuAppLinks.tsx`. Sorry for the confusion.

**Review by azangru — APPROVED** [review]

**Pull request merged**